java/gast_to_code/gast_to_code_java.py

```diff
@@ -27,7 +27,20 @@ def handle_log_statement(gast):
         return "System.out.println"
 
     def handle_var_assign(gast):
-        pass
+        var_id = router.gast_to_code(gast["varId"], "js")
```

Comment:
Is reusing the `js` helpers here intentional?

Reply:
+1 'js' should be reserved for javascript